src/sentry/monitors/consumers/monitor_consumer.py

```diff
@@ -51,19 +53,24 @@
 CHECKIN_QUOTA_LIMIT = 5
 CHECKIN_QUOTA_WINDOW = 60
 
-# This key is used when SENTRY_MONITORS_HIGH_VOLUME_MODE is enabled and we
-# trigger the monitor tasks as a side-effect of check-ins coming in. It is used
-# to store he last timestamp that the tasks were triggered.
-HIGH_VOLUME_LAST_TRIGGER_TS_KEY = "sentry.monitors.last_tasks_ts"
+# This key is used to store he last timestamp that the tasks were triggered.
+MONITOR_TASKS_LAST_TRIGGERED_KEY = "sentry.monitors.last_tasks_ts"
 
 
 class CheckinMessage(TypedDict):
+    # TODO(epurkhiser): We should make this required and ensure the message
+    # produced by relay includes this message type
+    message_type: NotRequired[Literal["check_in"]]
```

Comment:
Wondering if we'll really want to have this always in relay, it might be a bit of a weird pattern having messages in a topic that are of different format and are discriminated on a tag?

cc @wedamija @jan-auer

Reply:
also possible we could have some invalid setting (like `organization_id=-1` that just automatically tripped the killswitch and skipped any attempt to process it and keep it as more of a shell of a pseudo check-in. though that's much less explicit so not necessarily desirable as more of a hack

Reply:
Yeah, feels a good bit grosser though

Reply:
yeah just wanted to brainstorm/raise the idea